libraries/AP_Airspeed/AP_Airspeed_UAVCAN.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include "AP_Airspeed_Backend.h"
```

Comment:
Surround this header with HAL_WITH_UAVCAN or else surround the inclusion of this header in AP_Airspeed. Inclusion of UAVCAN headers will break the build. Travis is currently breaking due to this reason on your PR.

Reply:
sorry about this, do it right now.

Reply:
updated, but i always use "git push 'remote' HEAD --force" for updating code. what's the best way to do it?

Reply:
@liang-tang That is the correct way.